github/pulls_test.go

```diff
@@ -235,28 +236,58 @@ func TestPullRequestsService_Edit(t *testing.T) {
 	setup()
 	defer teardown()
 
-	input := &PullRequest{Title: String("t")}
+	tests := []struct {
+		input        *PullRequest
+		sendResponse string
 
-	mux.HandleFunc("/repos/o/r/pulls/1", func(w http.ResponseWriter, r *http.Request) {
-		v := new(PullRequest)
-		json.NewDecoder(r.Body).Decode(v)
+		wantUpdate   string
+		wantResponse *PullRequest
```

Comment:
nit: I find `wantResponse` a bit confusing here... what if we just called it `want` ?
Either way is fine, though... your call.